docs/start/hello-world.ipynb

```diff
@@ -118,6 +118,41 @@
     "IX = Pauli('IX')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "88fe43f8",
+   "metadata": {},
+   "source": [
+    "<Admonition type=\"note\" title=\"Important:\">\n",
+    "To ensure faster and more efficient results, as of 15 Feb 2024, circuits and observables need to be transformed to only use instructions supported by the system (referred to as *instruction set architecture (ISA)* circuits) before being submitted to the Qiskit Runtime primitives.  See the [transpilation documentation](../transpile) for instructions to transform circuits.  Due to this change, transpilation options (`options.transpilation.xxx`) are no longer supported and will be ignored. Additionally, `optimization_level`  no longer performs routing or layout operations.\n",
+    "</Admonition>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7a5af409",
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "",
+     "evalue": "",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[1;31mRunning cells with 'c:\\Python312\\python.exe' requires the ipykernel package.\n",
+      "\u001b[1;31mRun the following command to install 'ipykernel' into the Python environment. \n",
+      "\u001b[1;31mCommand: 'c:/Python312/python.exe -m pip install ipykernel -U --user --force-reinstall'"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager\n",
+    "\n",
+    "pm = generate_preset_pass_manager(backend=backend)\n",
+    "isa_circuit = pm.run(qc)\n",
+    "isa_observable = observable.apply_layout(isa_circuit.layout)"
+   ]
```

Comment:
@blakejohnson could you fix this?  I'm sure I don't have this observable modified correctly since they're defined differently than in the other examples.

Reply:
Done